lona/events/view_event.py

```diff
@@ -0,0 +1,24 @@
+from typing import TYPE_CHECKING, Optional, TypeVar, Union, Type, List, Dict
+
+if TYPE_CHECKING:
+    from lona.view import LonaView
+
+T = TypeVar('T', bound='LonaView')
+
+
+class ViewEvent:
+    def __init__(
+            self,
+            name: str,
+            data: Optional[Dict] = None,
+            view_classes: Optional[Union[Type[T], List[Type[T]]]] = None,
+            issuer: Optional[Union[T, str]] = None,
```

Comment:
Hm. The word "Optional" makes much sense to me in this context. Is there a conventional for this? I am ok with both

Reply:
1. I couldn't find any recommendations what is preferred.
2. There is a new syntax for type annotations. It allows to migrate from `t.Union[int, t.List[str]]` to `int | list[str]`. See [PEP 604](https://www.python.org/dev/peps/pep-0604/) and [PEP 585](https://www.python.org/dev/peps/pep-0585/).
3. I've just found out that it can be used in older python versions with `import __future__` (see [example](https://github.com/maratori/lona/commit/b5a8968f0c25d5d955e2bea9b46fd153547814f9)).
4. If you don't mind, I'd like to use new syntax in the project (I'll prepare PR). Any objections?
5. Going back to original question :) We have two options `None | T | str` vs `Optional[T | str]`. For me the first one is more consistent.
6. Another question rises: `Optional[str]` vs `None | str` vs `str | None`. Here I'm not pretty sure, but the second options wins in my head by a small margin.

Reply:
Yeah! ``None | T | str`` makes a lot of sense to me! I agree, and i am looking forward to a PR :)

Reply:
Here it is #138